docs/pages/database-access/guides/redis.mdx

```diff
@@ -0,0 +1,350 @@
+from clients that do not present a certificate.
+
+See [Configure TLS/SSL](https://redis.io/topics/encryption)
+in Redis documentation for more details.
+
+<Admonition type="note" title="Create Redis Cluster">
+  When you're configuring Redis Cluster you need to create the cluster using `redis-cli`
+
+  `redis-cli --cluster create` accepts only IP addresses, so the
+
+  Use below command to create the cluster:
+
+  ```shell
+  export REDISCLI_AUTH=STRONG_GENERATED_PASSWORD
+  export CERTS_DIR=/path/to/certs/
+
+  redis-cli --user alice --cluster-replicas 1 --tls --cluster-yes \
+  --cluster create ${IP1}:7001 ${IP2}:7002 ${IP3}:7003 ${IP4}:7004 ${IP5}:7005 ${IP6}:7006 \
+  --cacert ${CERTS_DIR}/server.cas --key ${CERTS_DIR}/server.key --cert ${CERTS_DIR}/server.crt
+  ```
+
+</Admonition>
+
+<Admonition type="note" title="Redis Cluster Only">
+
+  To enable Redis cluster mode in Teleport add `mode=cluster` parameter to connection uri in teleport config file.
+
+  ```yaml
+  databases:
+    - name: "redis-cluster"
+      uri: "rediss://redis.example.com:6379?mode=cluster"
+  ```
+
+</Admonition>
+
+## Step 5/5. Connect
+
+Log into your Teleport cluster and see available databases:
+
+```code
+$ tsh login --proxy=teleport.example.com --user=alice
+$ tsh db ls
+# Name          Description     Labels
+# ------------- --------------- --------
+# example-redis Example Redis env=dev
+```
+
+To connect to a particular database instance, first retrieve its certificate
+using `tsh db login` command:
+
+```code
+$ tsh db login example-redis
+```
+
+<Admonition type="tip" title="Tip">
+  You can be logged into multiple databases simultaneously.
+</Admonition>
+
+You can optionally specify the database name and the user to use by default
+when connecting to the database instance:
+
+```code
+$ tsh db login --db-user=alice example-redis
+```
+
+<Admonition type="note" title="Note">
+  If flag `--db-user` is not provided Teleport logs in as the `default` user.
+</Admonition>
+
+Once logged in, connect to the database:
+
+```code
+$ tsh db connect example-redis
+```
+
+<Admonition type="note" title="Note">
+  The `redis-cli` command-line client should be available in the system `PATH` in order to be
+  able to connect.
+</Admonition>
+
+Now you can log in as the previously created user using the below command:
+
+```
+AUTH alice STRONG_GENERATED_PASSWORD
+```
+
+To log out of the database and remove credentials:
+
+```code
+# Remove credentials for a particular database instance.
+$ tsh db logout example-redis
+# Remove credentials for all database instances.
+$ tsh db logout
+```
+
+### Supported Redis commands
+
+  <Tabs>
+    <TabItem label="Standalone">
+
+      Redis in standalone mode doesn't support below commands.
+
+      - HELLO
+      - PUNSUBSCRIBE
+      - SSUBSCRIBE
+      - SUNSUBSCRIBE
+
+      If one of the listed commands is called Teleport returns `ERR Teleport: not supported by Teleport` error.
```

Comment:
Let's add this to the paragraph before the list of commands so readers can have all the context they need first.

Reply:
Good idea, done.